feedback/models.py

```diff
@@ -55,3 +57,19 @@ class IssueChoices(models.TextChoices):
     entity_name = models.CharField(max_length=250)
     entity_url = models.CharField(max_length=250)
     data_custodian_email = models.CharField(max_length=250)
+
+    @property
+    def encoded_entity_url(self):
+        parsed_url: ParseResult = urlparse(self.entity_url)
+        encoded_path: str = quote(parsed_url.path)
```

Comment:
This seems ok as long as you've tested that the generated URLs still work ok 👍🏻 

It seems a bit odd to have to reencode here, as I was expecting this to be handled by django's `url` tag and `reverse` functions, which [encode the result](https://docs.djangoproject.com/en/5.1/ref/urlresolvers/#reverse:~:text=The%20string%20returned%20by%20reverse()%20is%20already%20urlquoted.%20For%20example%3A), but it seems like there is a difference in behaviour here

```python
>>> from urllib.parse import quote
>>> quote('foo(bar)')
'foo%28bar%29'
```

```python
>>> from django.utils.encoding import iri_to_uri
>>> iri_to_uri('foo(bar)')
'foo(bar)'
```

Apparently encoding parentheses is optional in a URI, so either should be valid normally.

Reply:
As discussed it is done here to avoid changing existing url behaviour. The only case we are catering for is when the url is sent to the notify service.